### DISCUSSION TITLE
Upgraded from 1.2.1 to 1.5.0 and nothing is working?

### DISCUSSION THREAD

**Discussion by avpengage** [garethgeorge/backrest#487] [Q&A]
I upgraded 2 different docker instances from 1.2.1 to 1.5.0, now neither instance will backup or do anything really.

Both instances use Backblaze as an S3 backend.

On the repo, Unlock appears to work (doesn't give an error, pbut likely doesn't actually work?), but check returns:

`Failed to check: : [unknown] check: check repo mbm2024: command "/bin/restic-0.17.0 check -o sftp.args=-oBatchMode=yes" failed: exit status 10`

index returns:

```
unknown] get snapshots for repo "mbm2024": get snapshots for repo mbm2024: command "/bin/restic-0.17.0 snapshots --json -o sftp.args=-oBatchMode=yes" failed: exit status 10
Output:
Fatal: repository does not exist: unable to open config file: stat s3.us-west-002.backblazeb2.com/config: no such file or directory
Is there a repository at the following location?
s3.us-west-002.backblazeb2.com
```
There is def a repository at that location, and it was backing up to it happily for 2+ months, until i just upgraded backrest...

Basically nothing is working - is there some simple fix I'm missing?

Any help or trouble shooting tips would be greatly appreciated.

Thanks!

**Comment by garethgeorge** [comment]
Continuing discussion in https://github.com/garethgeorge/backrest/issues/488
